index.d.ts

```diff
@@ -0,0 +1,47 @@
+declare module 'cerberus-node-client' {
+    interface CerberusClientOptions {
+        hostUrl: string;
+        region?: string;
+        token?: string;
+        debug?: boolean;
+    }
+
+    interface ListKeyResult {
+        keys: string[];
+    }
+
+    interface SecureFileSummaries {
```

Comment:
I don't think this name should not be plural, it describe a single instance.

Reply:
Tackled in most recent commit